subprojects/pcre2.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
+url = https://github.com/PCRE2Project/pcre2.git
+revision = 52c08847921a324c804cabf2814549f50bce1265
```

Comment:
Shouldn't we use release by default?

Reply:
Looks like they are preparing to make a new release soon, which would be great if done before our release: https://github.com/PCRE2Project/pcre2/commits/master/